Common/AM_datatypes.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+struct AM_SM_Commands {
+};
+
+struct SM_AM_Commands {
+    uint8_t motor_outputs[12];
+};
+
+struct PIDValues {
+    double P;
+    double I;
+    double D;
+};
+
+struct PIDControllerValues {
+    PIDValues axes[6];
+};
+
+struct GroundStationPIDSetCommand {
+    uint8_t controller;
+    uint8_t axis;
+    PIDValues values;
+};
+
+struct GroundStationPIDSetResponse {
+    uint8_t controller_number;
+    PIDControllerValues controller;
+};
```

Comment:
Need something to go from AM->Ground with pid values upon request. Not sure if it's here or later.

Reply:
I believe that we expect (for MVP) AM -> SM then SM-> TM messages that go to ground so that SM can insert the flight operation state.